Document result inconsistencies for getServicesMap (#92)
instana/openapi: spec/descriptions/getServicesMap.md

This endpoint retrieves services and connections (call paths) between them for calls in the scope given by the parameters.

> ## Result fields:

Reviewer: What do you think of changing the header to `Erratum`?

> - The `snapshotIds` field for result services is unsupported and always empty.

Reviewer 2: I would also prefer to have them ordered by their appearance in the OpenAPI result.

> - The `snapshotIds` field for result services is unsupported and always empty.

Author: updated

> - The `snapshotIds` field for result services is not used and always an empty list.

Reviewer: ```suggestion
- The `snapshotIds` field for result services is unsupported and always empty.
```

> - The `applications`, `maxSeverity` and `numberOfOpenIssues` fields for result services are not currently used and not present in the result JSON

Reviewer: ```suggestion
- The `applications`, `maxSeverity` and `numberOfOpenIssues` fields for result services are unsupported and always missing.
```
